cmd/samples/cron/main.go

```diff
@@ -1,66 +1,125 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"time"
 
 	"github.com/pborman/uuid"
+	"go.temporal.io/temporal-proto/serviceerror"
+	"go.temporal.io/temporal-proto/workflowservice"
 	"go.temporal.io/temporal/client"
 	"go.temporal.io/temporal/worker"
-
-	"github.com/temporalio/temporal-go-samples/cmd/samples/common"
+	"go.uber.org/zap"
 )
 
 const (
-	// ApplicationName is the task list for this sample
-	ApplicationName = "cronGroup"
+	// DomainName is the name of domain where workflow will be created.
+	DomainName = "samples-domain"
```

Comment:
No config. `DomainName` is hardcoded, and SDK default `hostPort` is used.

Reply:
Let's make domain optional in the sdks with defaulting it to "default". Then this is not necessary anymore.